the-murlocs/index.md

```diff
@@ -1,8 +1,50 @@
+During the formative years of the band it would have been accurate to describe King Gizzard and the Lizard Wizard as everyone's side project; now, they are the nexus for a range of solo and side acts, peers, collaborators, and even a new generation of bands who draw from their influence.
 
-<!-- TODO -->
-The Murlocs is a Garage/Blues Rock band fronted by Ambrose, featuring Cook on bass, and fleshed out with a semi-flexible crew from Melbourne. Formed around the same time as KGATLW, they have maintained a very respectable output between the many album releases and tours of their bigger and weirder counterpart. Their material is often a reflection of whatever Ambrose was contributing to KGATLW at the time, and is guaranteed to be enjoyed by any fan of his idiosyncratic voice and signature harmonica wail.
+## Music Projects Of King Gizzard Band Members
+
+Stu is the only member of King Gizzard not actively involved with composing music in other bands or solo projects (he does technical duties for some of these though):
+
+### The Murlocs  
+The Murlocs is a Garage/Blues Rock band fronted by Ambrose, featuring Cook on bass, with Cal Shortal, Tim Karmouche, and Matt Blach playing guitar keyboards and drums respectively. Formed around the same time as King Gizzard, they have maintained a very respectable output between the many album releases and tours of their bigger and weirder counterpart. Their material is a more straightforward reflection of whatever Ambrose was contributing to King Gizzard at the time, and is guaranteed to be enjoyed by any fan of his idiosyncratic voice and signature harmonica wail.
+
+### Pipe-Eye  
+This is the solo project of Cook, who began releasing music under the title in 2015. He typically writes and records all the music alone in his bedroom except drums, which are typically played by Cavs and recorded at King Gizzard headquarters. The music provides an insight into the existence of Cook Craig, who populates his songs with seemingly mundane topics delivered in a highly idiosyncratic, kooky style that will be recognisable to fans of the handful of King Gizzard songs he has written and recorded vocals for.
+
+### Bullant  
+Bullant is the solo techno/IDM production project of Joey. His debut album _Tyson, Crying_ shows the learning process of Ableton in real time, unable to save a project using the free version of the software, each song was completed in a single session under the stairs at the studio. Beginning in 2019, one could infer that this project was the offshoot of the electronic experiments of King Gizzard such as can be heard at the end of [_Fishing For Fishies_](https://kglw.net/releases/fishing-for-fishies), or in [_Made In Timeland_](https://kglw.net/releases/made-in-timeland).
+
+### CAVS  
+In 2021, Cavs released his debut solo self-titled drumming album. A pure expression of his drumming ability, CAVS shows off chops in various percussive styles outside of what he usually gets to play with King Gizzard.
```

Comment:
Wanna link to the Discogs page for these albums? e.g. https://www.discogs.com/master/2280727-Cavs-Cavs

Also should the album names all be italicized?

Reply:
"/associated-music" Sounds good. Discogs page and italicised titles also good, I'll get to those as a finish the body.